Packs/MicrosoftWindowsEvents/README.md

```diff
@@ -0,0 +1,42 @@
+# Microsoft Event Logs
+
+This pack includes Cortex XSIAM content.
+
+Note: The logs will be stored in the dataset named *microsoft_windows_raw*.
+
+To view logs only from the windows event log, please apply the following filter to the datamodel query: *| filter xdm.event.original_event_type="Microsoft-Windows-\*"*
+
+
+## Collect Events from Vendor
+
+In order to use the collector, you can use one of the following options:
+- [Microsoft Event Logs](#microsoft-event-logs)
```

Comment:
not sure why this line and the next one is here.  They are not options for using the collector.
I would only list XDRC (XDR Collector) and Broker VM.

Reply:
I agree, removed this extra line